packages/tabs/README.md

```diff
@@ -2,35 +2,76 @@
 
+[View the design documentation for this component.](https://spectrum.adobe.com/page/tabs/)
+
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tabs?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tabs)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tabs?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tabs)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-3smxuwr2)
 
-```
+```zsh
 yarn add @spectrum-web-components/tabs
 ```
 
 Import the side effectful registration of `<sp-tabs>`, `<sp-tab>` or `<sp-tab-panel>` via:
 
-```
+```ts
 import '@spectrum-web-components/tabs/sp-tabs.js';
 import '@spectrum-web-components/tabs/sp-tab.js';
 import '@spectrum-web-components/tabs/sp-tab-panel.js';
 ```
 
 When looking to leverage the `Tabs`, `Tab`, or `TabPanel` base class as a type and/or for extension purposes, do so via:
 
-```
-import {
-    Tabs,
-    Tab,
-    TabPanel
-} from '@spectrum-web-components/tabs';
+```ts
+import { Tabs, Tab, TabPanel } from '@spectrum-web-components/tabs';
 ```
 
-## Sizes
+### Anatomy
+
+- **Tabs:** The container component (`<sp-tabs>`) that manages the tab list and handles selection logic.
+- **Tab item:** An individual tab (`<sp-tab>`) that users can select to view different content panels.
+- **Tab view:** The content panel (`<sp-tab-panel>`) associated with a tab, shown when its corresponding tab is selected.
+- **Divider:** A visual separator between tab items, used in some variants for clarity.
+- **Selection indicator:** A visual highlight (such as an underline or bar) that shows which tab is currently selected.
+
+## Options
+
+<sp-table>
+    <sp-table-head>
+        <sp-table-head-cell>Property</sp-table-head-cell>
+        <sp-table-head-cell>Values</sp-table-head-cell>
+        <sp-table-head-cell>Default value</sp-table-head-cell>
+    </sp-table-head>
+    <sp-table-body>
+        <sp-table-row>
+            <sp-table-cell>label</sp-table-cell>
+            <sp-table-cell>text / nothing</sp-table-cell>
+            <sp-table-cell>–</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>icon</sp-table-cell>
+            <sp-table-cell>icon / nothing</sp-table-cell>
+            <sp-table-cell>nothing</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>is selected</sp-table-cell>
+            <sp-table-cell>yes / no</sp-table-cell>
+            <sp-table-cell>no</sp-table-cell>
+        </sp-table-row>
+        <sp-table-row>
+            <sp-table-cell>is disabled</sp-table-cell>
+            <sp-table-cell>yes / no</sp-table-cell>
+            <sp-table-cell>no</sp-table-cell>
+        </sp-table-row>
+    </sp-table-body>
+</sp-table>
+<br/>
+
+### Variants
```

Comment:
I am not sure this is correct. We have H2 on Options and H3 on Variants. What does the Options suggest? Can you please verify this if this is intended?